tutorial/09_sphinx/README.rst

```diff
@@ -224,3 +224,40 @@ Which looks like:
 Exercises
 ---------
 
+Generate Sphinx documentation on your own using the
+`pyvista/pyvista-doc-example <https://github.com/pyvista/pyvista-doc-example>`_
+repository. Either clone the respository with::
+
+  git clone https://github.com/pyvista/pyvista-doc-example
+
+Or simply download the repository:
+
+.. panels::
+
+   `PyVista Documentation Example <https://github.com/pyvista/pyvista-doc-example>`_
+
+   +++
+
+    .. link-button:: https://github.com/pyvista/pyvista-doc-example/archive/refs/heads/main.zip
+        :text: Download Zip
+        :classes: btn-outline-primary btn-block stretched-link
+
+
+Build the documentation
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Install the documentation build requirements with::
+
+  pip install -r requirements_docs.txt
```

Comment:
I think we need to explain explicitly that this is the `requirements_docs.txt` in the `pyvista-doc-example`. Installing the  `requirements_docs.txt` in the pyvista-tutorial repository takes 2 min in my environment.

Reply:
Good point. I'll fix that.

Reply:
Fixed in 8307cc7a4055e74e1371d5576775cd290fcf536d.